src/pages/documentation/consent-for-edge-network/api-reference.md

```diff
@@ -65,4 +65,27 @@ iOS
 
 <Tabs query="platform=ios&api=update-consents"/>
 
+<InlineAlert variant="info" slots="text"/>
+
+The updateConsents API automatically includes the following properties in its request to Edge Network: `standard` with the value `Adobe` and `version` with the value `2.0`. Additionally, the property `metadata` is set to the time at which the API is called.
```

Comment:
The Consent extension uses Adobe Standard 2.0 when communicating with the Edge Network, when the updateConsents API is called. Additionally, ...